### PR TITLE
[5.1] Directly call \Illuminate\Support\Arr functions

### PR DESCRIPTION
**Pull request by lucasmichot**
Directly call `\Illuminate\Support\Arr` functions

### CODE REVIEW

File: src/Illuminate/Foundation/Testing/CrawlerTrait.php [hunk 1]
```diff
@@ -249,11 +250,11 @@ protected function seeJsonContains(array $data, $negate = false)
             return $this->fail('Invalid JSON was returned from the route. Perhaps an exception was thrown?');
         }
 
-        $actual = json_encode(array_sort_recursive(
+        $actual = json_encode(Arr::sort_recursive(
             (array) $actual
         ));
 
-        foreach (array_sort_recursive($data) as $key => $value) {
+        foreach (Arr::sort_recursive($data) as $key => $value) {
```
Comment (crynobone) [inline src/Illuminate/Foundation/Testing/CrawlerTrait.php]: Does this method even exist?

Comment (lucasmichot) [inline src/Illuminate/Foundation/Testing/CrawlerTrait.php]: Typo fixed

**Comment by lucasmichot** [comment]
Not broken, see https://github.com/laravel/framework/pull/10621

**Comment by GrahamCampbell** [comment]
I didn't ask you to close all your PRs. Closing them is up to you.

File: src/Illuminate/Routing/Router.php [hunk 2]
```diff
@@ -404,7 +404,7 @@ public static function mergeGroup($new, $old)
             $new['as'] = $old['as'].(isset($new['as']) ? $new['as'] : '');
         }
 
-        return array_merge_recursive(array_except($old, ['namespace', 'prefix', 'where', 'as']), $new);
+        return array_merge_recursive(Arr::except($old, ['namespace', 'prefix', 'where', 'as']), $new);
```
Comment (crynobone) [inline src/Illuminate/Routing/Router.php]: Using array_* for normal one time operation is fine, however this is going to be hit whenever you use recursive `Route::group()`, I would prefer it it's avoid having to use alias.

Comment (GrahamCampbell) [inline src/Illuminate/Routing/Router.php]: I agree.

**Comment by crynobone** [comment]
@lucasmichot some of this has been merged before.

See #9008 and #9011

**Comment by GrahamCampbell** [comment]
Yeh. I'm :+1: for merging this one.

**Comment by lucasmichot** [comment]
Sure @GrahamCampbell - they are not fix

**Comment by lucasmichot** [comment]
(fixes I mean)

**Comment by GrahamCampbell** [comment]
It's just a case of being sensible with what you change.

**Pull request closed without merging**